### ISSUE TITLE
missing generated source map?

### ISSUE THREAD

**Issue by imWildCat** [Bram-dc/vite-plugin-react-native-web#2]
Recently I started to upload my source maps to Sentry.

However, it is showing:

```
[plugin react-native-web] Sourcemap is likely to be incorrect: a plugin (react-native-web) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
```

any hints to fix this? I can create a PR if needed

**Comment by Bram-dc** [comment]
@imWildCat Could you check if this is resolved in the latest version 1.1.1?

**Comment by imWildCat** [comment]
Hi @Bram-dc sorry for the delay!

I was traveling : )

I can confirm this problem is fixed now. Appreciate your quick action here!
